GitUI/UserControls/RevisionGrid/RevisionGridRefRenderer.cs

```diff
@@ -22,11 +22,11 @@ public static void DrawRef(bool isRowSelected, Font font, ref int offset, string
             var paddingTopBottom = DpiUtil.Scale(2);
             var marginRight = DpiUtil.Scale(7);
 
-            var textColor = fill ? headColor : Lerp(headColor, Color.White, 0.5f);
+            var textColor = fill ? headColor : Lerp(headColor, Color.White, 0.0f);
 
             var textSize = TextRenderer.MeasureText(graphics, name, font, Size.Empty, TextFormatFlags.NoPadding);
 
-            var arrowWidth = arrowType == RefArrowType.None ? 0 : bounds.Height / 2;
+            var arrowWidth = arrowType == RefArrowType.None ? 0 : (bounds.Height / 2) - 2;
```

Comment:
Do we have screenshots of HEAD and new with such arrows?

Reply:
Current on top of page.

Reply:
Removed this change. For the following researchers. 😉